addons/api/mirage/generated/factories/target.js

```diff
@@ -1,16 +1,12 @@
 import { Factory } from 'ember-cli-mirage';
 import { faker } from '@faker-js/faker';
-import { TYPE_TARGET_TCP } from 'api/models/target';
 
 /**
  * GeneratedTargetModelFactory
  */
 export default Factory.extend({
-  type: TYPE_TARGET_TCP,
```

Comment:
The default type is TCP.

Reply:
This can actually be removed since we are setting the type dynamically in the target factory. I will remove this line.